tools/lou_checkyaml.c

```diff
@@ -406,10 +484,18 @@ read_options(yaml_parser_t *parser, int len, int *xfail, translationModes *mode,
 			*mode = read_mode(parser);
 		} else if (!strcmp(option_name, "typeform")) {
 			yaml_event_delete(&event);
-			*typeform = read_typeforms(parser, len);
-		} else if (!strcmp(option_name, "cursorPos")) {
+			*typeform = read_typeforms(parser, wordLen);
+
+			// FIXME: Comparisons should be truly case insensitive, but apparently no standard function for that.
```

Comment:
I would personally prefer that the option names are not case insensitive at all.

Reply:
There were already some inconsistancy between cursorPos (cap P) and inpos and outpos in other contexts (small p). You could say that cursor position is clearly two words, while inpos and outpos are not necesarily two words. Nevertheless, it is a potential source of confusion. So, I thought we should be lenient on cases. Either that or stick to lower case completely.

Reply:
Let's just call it "inputPos", "outputPos" and "cursorPos" for now. Hopefully we can abstract out inputPos and outputPos in the future anyway. And eventually when cursorPos is only a single number I'd like to rename it to simply "cursor" ("pos" is kinda redundant).

Reply:
Agreed